recipes/optionsfactory/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "optionsfactory" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43f0c4ace9ecde6e755aaeef0bcd9ff78c2503e85d189c6db51c7b8387183819
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
```

Comment:
```suggestion
```

Reply:
does your package actually need pip?

Reply:
No, it doesn't actually need pip. Sorry, that line was just copied from the example!

Reply:
It does. `pip` is used for building this package.

Reply:
:+1: Thanks @xhochy